workers/generic-worker/fileutil/fileutil_windows.go

```diff
@@ -16,3 +19,15 @@ func SecureFiles(filepaths ...string) (err error) {
 	}
 	return
 }
+
+func ResetPermissions(path string, permissions fs.FileMode) error {
+	return host.Run("icacls", path, "/reset", "/t")
+}
+
+func GetPermissionsString(path string) (string, error) {
+	return host.CombinedOutput("icacls", path)
+}
+
+func GetPermissions(path string) (fs.FileMode, error) {
+	return 0, fmt.Errorf("GetPermissions not implemented on Windows")
+}
```

Comment:
If it isn't implemented, why do we include it?

Reply:
Because the multiuser test is calling GetPermissions so wouldn't there be a compilation error on windows if this wasn't defined?